day7/sheldon/index.css

```diff
@@ -0,0 +1,410 @@
+@font-face {
+  font-family: "ttnorms";
+  src: url(ttnorms/medium.woff);
+}
+
+body {
+  margin: 0;
+  background: linear-gradient(rgb(67, 67, 175), rgb(187, 84, 187));
+  font-family: "ttnorms", Fallback, sans-serif;
+}
+
+.page {
+  height: 100vh;
+  width: 100%;
+  overflow: hidden;
+}
+
+.mobile__content {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  text-align: center;
+  padding: 48px 30px 40px 30px;
+  box-sizing: border-box;
+  z-index: 2;
+  position: relative;
+}
+
+.mobile__header {
+  display: flex;
+  flex-direction: column;
+  color: white;
+}
+
+.mobile__icon svg {
+  padding: 32px 17px;
+  border-radius: 30px;
+  border: 1px solid rgba(255, 255, 255, 0.2);
+}
+
+.mobile__title {
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 35px;
+  margin-top: 16px;
+}
+
+.download {
+  color: white;
+  background: #ea3f58;
+  box-shadow: 0px 3px 15px rgba(9, 65, 125, 0.15);
+  border-radius: 41px;
+  padding: 22px 0px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  border: none;
+}
+
+img {
+  width: 822px;
+  transform: rotate(-20deg) translateX(65px) translateY(-680px);
+  z-index: 1;
+}
+
+@media only screen and (min-width: 768px) {
+  .page {
+    height: auto;
+  }
+  .mobile__container {
+    display: none;
+  }
+
+  .portrait__container {
+    color: white;
+  }
+
+  .logo {
+    padding: 32px 25px 42px 25px;
```

Comment:
use padding only on top and left corner, the bottom padding/margin should be determined by the element below

Reply:
fixed